src/barkhaPJun20/MenuDriven.java

```diff
@@ -0,0 +1,215 @@
+package barkhaPJun20;
+
+import java.util.Scanner;
+
+public class MenuDriven {
+	static double balance;
+	static int totalcreditcount;
+	static int totaldebitcount;
+	int creditcount, debitcount;
+	String bank;
+	void bankOperation(String bankName, double amount, String operation) {
+		bank=bankName;
+		switch (bankName) {
+			case "SBI":
+			switch (operation)	{
+				case "Credit":
+				balance=balance+amount;
+				totalcreditcount++;
+				creditcount++;
+				break;
+				case "Debit":
+					if (balance>=amount) {
```

Comment:
@BarkhaPJun20 
Check condition 
What you think it should be greater than or less than